### PR TITLE
Add checkDefault

### PR DESCRIPTION
**Pull request by Yeti-or**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,9 @@
     "node": ">= 4.0.0"
   },
   "scripts": {
-    "test": "jscs ./ && jshint . && mocha --recursive"
+    "test": "npm run lint && npm run specs",
```
Comment (qfox) [inline package.json]: "pretest": "npm run lint",
"test": "npm run specs" ?

Comment (Yeti-or) [inline package.json]: `pretest` is official?

Comment (qfox) [inline package.json]: https://docs.npmjs.com/misc/scripts

Yup. preANY as well as postANY

Comment (Yeti-or) [inline package.json]: fucking cool. yes it is!

**Pull request merged**